### ISSUE TITLE
Is there any way to change the space taken by the legends in a chart ?

### ISSUE THREAD

**Issue by Saroopashree** [jerairrest/react-chartjs-2#516]
![Pie](https://user-images.githubusercontent.com/47754198/81826514-7ec7bf00-9555-11ea-9a57-40072a69e85d.png)

**Comment by Ahmad-AlShalabi** [comment]
Yes, there is, you can change the box size of it  and the padding between the legends:
 `var chartLegend = {`
     ` labels: {`
      `  boxWidth: 15,`
     `   padding: 10,   },  }; `
`  <Pie legend={chartLegend} data={chartData}></Pie>`

**Comment by Saroopashree** [comment]
When changing the value for the `boxWidth`, only the width of the colored boxes in each of the legend items are changed.
Adding `padding` as you have mentioned creates padding between each of the rows in the legend set.

But the desired behavior is to set a max-height property to limit the space taken by the legend section.
Is there any way we can achieve this behavior @Ahmad-AlShalabi  ?

**Comment by morucci** [comment]
I'm facing the exact same issue. What could be the solution ?
